Source/HTTP/Curl/CurlEasyRequest.cpp

```diff
@@ -18,13 +18,32 @@ CurlEasyRequest::CurlEasyRequest(CURL* curlEasyHandle, HCCallHandle hcCall, XAsy
 
 CurlEasyRequest::~CurlEasyRequest()
 {
+#if HC_PLATFORM == HC_PLATFORM_GDK
+    if (CurlDynamicLoader::GetInstance().IsLoaded())
+    {
+        CURL_CALL(curl_easy_cleanup)(m_curlEasyHandle);
```

Comment:
>CURL_CALL [](http://example.com/codeflow?start=8&length=9)

To make code more Xplat couldn't you just define CURL_CALL to be basically a no-op on non GDK platforms? you'd need many less if-defs throughout the code then

Reply:
You could also add something like

CONDITIONAL_CURL_CALL(...)

that first validates the library is loaded on GDK and does nothing on non-gdk